modules/statistics/test/TestPlan.md

```diff
@@ -38,3 +38,28 @@
 5. Check that `Show/Hide` visit labels works.
 6. Check that changing the breakdown adjusts the table accurately.
 7. Verify site and grand totals.
+
+### Registration of widgets on the dashboard page
+1. Make sure there is no recruitment target set in the configuration module. Check that an incentive to define a
+   recruitment target is displayed in the recruitment panel of the dashboard page.
+   [Automate Test on Travis_CI]
```

Comment:
what's the difference between "Automate Test on Travis_CI" and "Automate Test"?

Reply:
I removed ' on Travis_CI'.